server/service/integration_core_test.go

```diff
@@ -333,7 +333,7 @@ func (s *integrationTestSuite) TestVulnerableSoftware() {
 
 	lsReq := listSoftwareRequest{}
 	lsResp := listSoftwareResponse{}
-	s.DoJSON("GET", "/api/v1/fleet/software", lsReq, http.StatusOK, &lsResp, "vulnerable", "true")
+	s.DoJSON("GET", "/api/v1/fleet/software", lsReq, http.StatusOK, &lsResp, "vulnerable", "true", "order_key", "generated_cpe", "order_direction", "desc")
```

Comment:
We need to test something that returns `> 1` elements so you can check the ordering is correct.

Reply:
This is mainly testing the decoding of the key. Ordering is (or should) be tested at the datastore level.